engines/hypno/lexer_arc.cpp

```diff
@@ -280,6 +280,7 @@
 #include <string.h>
 #include <errno.h>
 #include <stdlib.h>
+#include <stdint.h>
```

Comment:
Same thing, this file is autogenerated, no sense to edit it. If you need to add something, the proper place is .l file